wt-config.cmake.in

```diff
@@ -1,3 +1,21 @@
+include(CMakeFindDependencyMacro) 
```

Comment:
Why we have `wt-config.cmake.in` and `cmake/Config.cmake.in`? One of them not used really?